ducktape/cluster/remoteaccount.py

```diff
@@ -260,21 +255,24 @@ def ssh(self, cmd, allow_fail=False):
         :return: The exit status of the command.
         :raise RemoteCommandError: If allow_fail is False and the command returns a non-zero exit status
         """
-        self._log(logging.DEBUG, "Running ssh command: %s" % cmd)
+        self.logger.debug("Running ssh command: %s" % cmd)
 
         client = self.ssh_client
         stdin, stdout, stderr = client.exec_command(cmd)
 
-        # Unfortunately we need to read over the channel to ensure that recv_exit_status won't hang. See:
-        # http://docs.paramiko.org/en/2.0/api/channel.html#paramiko.channel.Channel.recv_exit_status
-        stdout.read()
+        stdout_contents = stdout.read().strip()
+        stderr_contents = stderr.read().strip()
+        if stdout_contents:
```

Comment:
Do we want this here? It's going to add a lot of noise since this will log the output of every command, regardless of whether successful. Seems like putting it in the final non-failure allowed block (line 275) covers the 99% use case?

Reply:
FWIW, i've really been enjoying having all this additional output. we should also consider going through services and get rid of the many areas people explicitly log this functionality